README.md

```diff
@@ -13,27 +13,57 @@ npm install --save-dev @faker-js/cli
 ## Usage
 
 ```
-faker module_name method_name
+npx faker <moduleName> <functionName>
 ```
 
-Faker-CLI expects a `module_name` as well as a `method_name` depending on the module chosen.
+> [ℹ: Read more about `npx`](https://docs.npmjs.com/cli/v7/commands/npx)
 
-To list all possible modules run:
+Faker-CLI expects a `moduleName` as well as a `functionName` argument.
+This is equivalent to calling `faker[moduleName][functionName]`.
+Checkout [Fakers's API](https://fakerjs.dev/api/) for information on which modules and functions are available.
 
-```bash
-faker --help
-```
+## Examples
 
-To list all possible methods of a module run:
+If you want a integer you can run:
 
 ```bash
-faker module_name --help
+npx faker number int
 ```
 
 ## Whats next
 
 Upcoming features might include:
 
-- localization
-- support for all parameters `@faker-js/faker` natively
-- make the CLI consumable for your own modules
+### Localization
+
+You should be able to provide a specific locale for localized data.
```

Comment:
Maybe put a `🛠` icon at the front of the first paragraph for a feature that is not yet implemented?

Reply:
That a neat idea. Makes it more clear we are in the "under development" section. 👍